packages/widgets-mitosis/src/components/widgets/Accordion/accordion-item.lite.tsx

```diff
@@ -0,0 +1,109 @@
+import { Show, useStore, useContext, For } from '@builder.io/mitosis';
+// these elements needs to be migrated from @builder.io/sdks
+import { BuilderElement } from '../../../types/element';
+import RenderBlocks from '../../render-blocks.lite';
```

Comment:
where is this coming from?

Reply:
If this is reaching into `packages/sdks` folder, can we add a TODO comment with steps on how we plan to migrate?

Reply:
No these are just pseudo imports, Not linked to mitosis sdks currently. We will be able to import these once @builder.io/sdks is available to install and exports the required components.